documentation/docs/verifiable_credentials/create.mdx

```diff
@@ -12,15 +12,59 @@ keywords:
+- [**Schema**](https://www.w3.org/TR/vc-data-model/#data-schemas): optional list of objects specifying the ID and Type of the schema that the data must conform to. 
+- [**Refresh Service**](https://www.w3.org/TR/vc-data-model/#refreshing): optional link to a service where the recipient may refresh the included credentials.
+- [**Terms of Use**](https://www.w3.org/TR/vc-data-model/#terms-of-use): optional list of policies defining obligations, prohibitions, or permissions of the presentation recipient.
+- [**Evidence**](https://www.w3.org/TR/vc-data-model/#evidence): optional list of objects specifying a Type that can be used by the issuer to provide the verifier with additional supporting information in a verifiable credential. 
+- [**Non Transferable**](https://www.w3.org/TR/vc-data-model/#nontransferable-property): optional flag that indicates that a verifiable credential must only be encapsulated into a verifiable presentation whose proof was issued by the credentialSubject.
 
-## Example
+After creation, the issuer signs the verifiable credential using a private key, which will allow verifiers to validate the VC with its public key.
+
+### Proof Options {#proof-options}
+
+A digital signature on a verifiable credential both provides data integrity and proves the DID of the issuer.
+The proof section embedded in a credential may also include additional metadata.
+
+The following metadata properties can be configured by the framework and are optional and omitted by default:
+
+- **Created**: timestamp of when the credential was signed.
+- **Expires**: timestamp after which the credential is no longer considered valid.
+- **Challenge**: arbitrary random string. Sent by the issuer and mitigates replay attacks; should be sufficiently random and uniquely generated per credential request.
+- **Domain**: arbitrary string. Sent by the issuer and can help mitigate replay attacks when used with a challenge.
+- **Proof Purpose**: indicates the purpose of the signature.
+  - **AssertionMethod**: to assert a claim. The signing verification method must have an [`assertionMethod`](https://www.w3.org/TR/did-core/#assertion) relationship to be valid.
+  - **Authentication**: to authenticate the signer. The signing verification method must have an [`authentication`](https://www.w3.org/TR/did-core/#authentication) relationship to be valid.
+
+### Credential Validation {#credential-validation}
 
-This example shows how you can create a Verifiable Credential and validate it. In this example, Alice takes the role of the subject, while we also have an issuer.
-The issuer signs a UniversityDegreeCredential type verifiable credential with Alice's name and DID.
-This Verifiable Credential can be verified by anyone, allowing Alice to take control of it and share it with anyone.
+After creating a verifiable credential, and before sending the VC to the subject, an issuer may want to make sure certain properties of the VC are valid. 
```

Comment:
Excellent addition, might be good to mention that the Subject / Holder might also want to do this to make sure they received a valid VC. And could at any point do this to see if their VCs are still valid.